.github/workflows/velox_be.yml

```diff
@@ -511,7 +520,7 @@ jobs:
       - name: Setup docker container
         run: |
           docker run --rm --init --privileged --ulimit nofile=65536:65536 --ulimit core=-1 --security-opt seccomp=unconfined \
-          -v $PWD:/opt/gluten --name static-build-test-$GITHUB_RUN_ID -e NUM_THREADS=30 -detach $DOCKER_REGISTRY_HOST:$DOCKER_REGISTRY_PORT/gluten-dev/centos:7 \
+          -v $PWD:/opt/gluten --name static-build-test-$GITHUB_RUN_ID -e NUM_THREADS=30 -detach $PREVIOUS_DOCKER_PUSH_REGISTRY/gluten-dev/centos:7 \
```

Comment:
So we are not able to use the scripts for `static-build-test`? Do you know some reasons?

Reply:
This involves the incompatibility of the git version in the new centos7 image. 

I encountered network problems while trying to fix it, so put this problem aside and use the previous image.